projects/ad9656/zcu102/system_bd.tcl

```diff
@@ -0,0 +1,23 @@
+
+
+## NOTE: With this configuration the #36Kb BRAM utilization is at ~57%
+
+source $ad_hdl_dir/projects/common/zcu102/zcu102_system_bd.tcl
+
+#system ID
+ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
+ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "[pwd]/mem_init_sys.txt"
+ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
+set sys_cstring "sys rom custom string placeholder"
+sysid_gen_sys_init_file $sys_cstring
+
+ad_mem_hp0_interconnect sys_cpu_clk sys_ps8/S_AXI_HP0
+
+source ../common/ad9656_bd.tcl
+
+ad_ip_parameter axi_ad9656_rx_dma CONFIG.DMA_DATA_WIDTH_DEST 128
```

Comment:
Given that we're only supporting ZCU102, these parameters should be in the common file

Reply:
Parameters moved to cmmon file